libraries/AP_Scripting/applets/Aerobatics/FixedWing/TrajectoryBased/Schedules/README.md

```diff
@@ -0,0 +1,35 @@
+# Scripted Aerobatics Schedules
+
+This directory contains full aerobatic schedules for the ArduPilot
+fixed wing scripted aerobatics system. If you develop your own
+schedules please contribute to the project by submitting them for
+inclusion in this directory.
+
+## Usage
+
+To use one of these schedules in SITL copy it (or use a symbolic link)
+to a file called trickNN.txt in your scripts/ directory (the same
+directory that plane_aerobatics.lua goes in). The NN is the trick
+number used in the AUTO mission or with tricks on a switch in the
+TRIKn_ID parameter.
+
+For example, if you wanted to fly the NewZealand Clubman schedule you
+could copy NZClubMan.txt to trick94.txt in your scripts
```

Comment:
Why trick 94? This numbering seems to be important, but nothing I've seen explains why it should be 94 and not some other number.

Reply:
historical from previous commits...not mine...but will take the opportunity to fix